src/test/java/com/epam/training/student_gregory_kubya/CartPage.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.training.student_gregory_kubya;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindAll;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class CartPage {
+
+  private final WebDriver driver;
+  private static final String PAGE_URL = "https://fiftyoutlet.com/es/es/cart";
+
+
+  @FindAll({@FindBy(css = "div.name.product-name")})
+  List<WebElement> cartNameList;
+  @FindAll({@FindBy(css = "div.sku.product-ref")})
+  List<WebElement> cartReferenceList;
+  @FindAll({@FindBy(css = ".price-total")})
+  List<WebElement> cartPriceList;
+  @FindAll({@FindBy(css = ".item-quantity>.input-text")})
+  List<WebElement> cartQuantityList;
+
+  @FindBy(css = ".order-subtotal>.order-totals-value")
+  private WebElement orderTotal;
+
+
+  public CartPage(WebDriver driver) {
+    this.driver = driver;
+    PageFactory.initElements(driver, this);
+  }
+
+
+  public void openCartPage() {
+    driver.get(PAGE_URL);
+  }
+
+  public List<String> productNamesOnCart() {
+    return cartNameList.stream()
+        .map(WebElement::getText)
+        .toList();
+  }
+
+  public double getOrderTotal() {
+    return Double.parseDouble(orderTotal.getText()
+        .replaceAll("\\s|€", "").replace(",", "."));
+  }
+
+  public ArrayList<ProductDTO> cartProducts() {
```

Comment:
I suggest using List type of collection and clarifying in the function name what we do.. getProductListFromCartPage?